docs/Releasing.adoc

```diff
@@ -0,0 +1,58 @@
+= Releasing Kroxylicious
+3.0, July 29, 2022: AsciiDoc article template
+:toc:
+:icons: font
+:url-quickref: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/
+
+A quick and dirty guide to actually releasing builds for consumption via maven central.
+
+== Requirements
+.Maven Central
+* A Maven Central/Sonatype account. Accounts are created via their https://issues.sonatype.org/secure/Signup!default.jspa[JIRA signup]
+* Your Sonatype account needs to be granted publication permissions for the Kroxylicous project.
+** One of the existing team members with publishing rights needs to raise a https://issues.sonatype.org/secure/CreateIssue.jspa?pid=10134&issuetype=11003[JIRA ticket] to request the new account is added (or removed...). The requirements for the ticket are listed in the https://central.sonatype.org/publish/manage-permissions/[Central Docs]
+
+Signing the builds with a GPG key is a requirement for publishing to Maven Central.
+
+.PGP
+* Ensure your PGP key is published to a public key server (for example keys.openpgp.org)
+* Get your PGP key signed by the Kroxylicious project key. Rather than share a single key between project members (which has issues with revocation and re-issuing) we sign the releases individually using a key signed by the project key as a mark of trust.
+* A copy of your keys short ID.
+
+TIP: To get the short ID for your key `gpg2 --show-keys --keyid-format short $\{YOUR_KEY_LONG_ID}`
+
+== Running a release
+
+.Releasing Locally
+. `mvn clean install`
+.. We are looking for a clean and green build.
+.. If it's not, shout scream and swear.
+.* Think *very very* carefully if its worth proceeding. Releases are forever!
+. Switch to the release version number.
+.. `export RELEASE_VERSION=0.1.0`
+.. `mvn versions:set -DnewVersion=$\{RELEASE_VERSION}`
```

Comment:
After this step the property is:
```
        <kroxyliciousApiVersion>0.1.0-SNAPSHOT</kroxyliciousApiVersion>
```
but the api artifacts are at `0.1.0`

Reply:
Hmm good spot.

Reply:
fixed in 9d02b10a8854c6732538e85b0748064f8d2a9f5f